core/server/worker/src/main/java/alluxio/worker/block/reviewer/ProbabilisticReviewer.java

```diff
@@ -0,0 +1,56 @@
+package alluxio.worker.block.reviewer;
+
+import alluxio.conf.PropertyKey;
+import alluxio.conf.ServerConfiguration;
+import alluxio.worker.block.meta.StorageDirView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Random;
+
+/**
+ * Reviews a block allocation, and probably reject a bad block allocation.
+ * The more full a StorageDir, the higher the chance that this reviewer decides
+ * not to use it for a new block.
+ * */
+public class ProbabilisticReviewer implements Reviewer {
+    private static final Logger LOG = LoggerFactory.getLogger(ProbabilisticReviewer.class);
+
+    // TODO(jiacheng): Make this a property
+    private static final double CUTOFF = 0.1;
```

Comment:
0.1 is probably too big, like when we have 1TB for a storage dir, that is too far from a block size. 
64MB / 1TB = 0.00006. I don't think we want to assign this small decimal to a property.
If a `long` in bytes make more sense, maybe we can reuse `PropertyKey.WORKER_TIERED_STORE_FREE_AHEAD_BYTES`?

`WORKER_TIERED_STORE_FREE_AHEAD_BYTES` is a property we added in 2.3 together with the new CACHE_PROMOTE. I don't think many people use that. I also don't think the property is very useful at the moment. It only controls how much extra we free, on each `TieredBlockStore.freeSpace`.

Reply:
Done. Please see if the new properties make sense. Thanks!